gradle-plugins/rnbrownfield-gradle-plugin/example-android-library/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+    id("com.callstack.rnbrownfield-plugin")
```

Comment:
could we go with something like `com.callstack.react:brownfield` ? No other plugin ids use "-plugin" suffix.

Reply:
We can't apply a plugin with `:` in between. It is not allowed. We can use something like `id("com.callstack.react.brownfield")` and in class path we can do `classpath("com.callstack.react:brownfield:0.1.0")`

I have addressed this change and will commit shortly